apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance_row.jsx

```diff
@@ -25,13 +25,18 @@ const SessionAttendanceRow = React.createClass({
       enrollment_id: React.PropTypes.number.isRequired,
       user_id: React.PropTypes.number,
       in_section: React.PropTypes.bool.isRequired,
-      attended: React.PropTypes.bool.isRequired
+      attended: React.PropTypes.bool.isRequired,
+      puzzles_completed: React.PropTypes.number.isRequired
     }).isRequired,
```

Comment:
Hmm, these all come from the Rails [serializers](https://github.com/code-dot-org/code-dot-org/pull/15227/files#diff-df76b72f2fef6f4ef3614482a9595f37R21), which in the default case just refer to a property on the Rails model, which is underscored.